Resources/Prototypes/_RMC14/Entities/Clothing/Eyes/glasses.yml

```diff
@@ -119,3 +119,15 @@
     sprite: _RMC14/Objects/Clothing/Eyes/Glasses/Prescription_ballistic_goggles.rsi
   - type: Clothing
     sprite: _RMC14/Objects/Clothing/Eyes/Glasses/Prescription_ballistic_goggles.rsi
+
+- type: entity
+  parent: ClothingEyesBase
+  id: RMCWeldingGoggles
+  name: Welding goggles
+  description: A pair of welding goggles. Provides protection from eye damage.
+  components:
+  - type: Sprite
+    sprite: _RMC14/Objects/Clothing/Eyes/Glasses/welding_goggles.rsi
+  - type: Clothing
+    sprite: _RMC14/Objects/Clothing/Eyes/Glasses/welding_goggles.rsi
+  - type: EyeProtection
```

Comment:
These need to be toggleable (helmet too one day) to be made effective but lower your vision range like in 13, I don't think we have anything that enables that yet but I can do that later